packages/suite/src/views/suite/SwitchDevice/DeviceItem/DeviceHeader.tsx

```diff
@@ -47,11 +58,21 @@ export const DeviceHeader = ({
             )}
 
             {deviceModelInternal && isDeviceStatusVisible && (
-                <DeviceStatus
-                    deviceModel={deviceModelInternal}
-                    device={device}
-                    forceConnectionInfo={true}
-                />
+                // In case the cancel button is not visible (e.g. in device switcher),
+                // we need to have an alternative, mostly for testing purposes.
+                <Box
+                    data-testid={
+                        isDefaultCancelVisible ? undefined : '@switch-device/cancel-button'
```

Comment:
Looks like default cancel has no testid even though the test id sounds like cancel button. Confusing  🤔